chain/network/Cargo.toml

```diff
@@ -7,9 +7,20 @@ edition = "2021"
 rust-version = "1.60.0"
 publish = false
 
+[build-dependencies]
+prost-build = { version = "0.10"}
+sha2 = "0.9"
```

Comment:
Could use near_stable_hasher here given that we already have it.

Reply:
It's not about stability, it's about not having logical duplicates. We *already* use `near_stable_hasher` for `near-network` so there's no need to add sha2 on top of that. For this use-case, stable-hasher and sha2 play equivalent roles -- provide some hash you can use for fingerprinting.